sql/records.cc

```diff
@@ -664,8 +663,7 @@ static int rr_from_cache(READ_RECORD *info)
       int3store(ref_position,(long) i);
       ref_position+=3;
     }
-    my_qsort(info->read_positions, length, info->struct_length,
-             (qsort_cmp) rr_cmp);
+    my_qsort(info->read_positions, length, STRUCT_LENGTH, (qsort_cmp)rr_cmp);
```

Comment:
According to our coding style there should be space between cast and variable: `(qsort_cmp) rr_cmp)`.

Reply:
Is coding style written on some web page?